src/Kunstmaan/GeneratorBundle/Generator/PagePartGenerator.php

```diff
@@ -95,7 +96,7 @@ private function generatePagePartEntity()
 
         $this->assistant->writeLine('Generating entity : <info>OK</info>');
     }
-
+    
```

Comment:
don't do code formatting of old code in a feature PR, als think that there should not be spaces here